src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -56,7 +56,7 @@ public void execute_invalidCommandFormat_throwsParseException() {
 
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
-        String deleteCommand = "delete 9";
+        String deleteCommand = "delete 7";
```

Comment:
Please explain this? 🤔

Reply:
This is a mistake made. Was testing other values as test case was failing, fixed the error at the other test cases